contracts/cheddar_5x5_tic_tac_toe/src/board.rs

```diff
@@ -52,13 +52,14 @@ impl Board {
             tiles: UnorderedMap::new(StorageKey::GameBoard { game_id }),
             current_piece: Piece::O,
             winner: None,
+            last_move: None,
         }
     }
     pub fn check_move(&self, coords: &Coords) -> Result<(), MoveError> {
         if self.winner.is_some() {
             return Err(MoveError::GameOver);
         }
-        if coords.y >= BOARD_SIZE || coords.x >= BOARD_SIZE {
+        if coords.y >= BOARD_SIZE as u8 || coords.x >= BOARD_SIZE as u8 {
```

Comment:
can we keep `BOARD_SIZE` as `u8` and then we can reduce most of the `as` convertions. (including the for loop)